src/playground-ide.ts

```diff
@@ -91,6 +108,30 @@ export class PlaygroundIde extends LitElement {
     slot {
       display: none;
     }
+
+    #resizeBar {
+      position: absolute;
+      top: 0;
+      left: -5px;
+      width: 10px;
+      height: 100%;
+      z-index: 9;
+      cursor: col-resize;
```

Comment:
I think it'd be nice to have a default hover-over highlighting here. I find that it feels better when the cursor doesn't just change on otherwise unindicated regions

Reply:
Can you clarify? Do you mean that hovering over the resize bar should change the color of something, but not change the cursor? Which element's color should change?